### DISCUSSION TITLE
Guide on how to activate and wire the fake power switch on IO13

### DISCUSSION THREAD

**Discussion by WMasterJedi** [CircuitSetup/Time-Circuits-Display#19] [Q&A] [answered]
I would like to know how to wire a switch onto IO13 and how to activate the feature.

**Comment by CircuitSetup** [comment, marked as answer]
I will add this to the Wiki shortly

1. On the back of the keypad is a space for IO13 - Install a switch between IO13 and GND so that switching it on connects the two. I would recommend soldering on a 3 pin header (2.54mm spacing)
2. In the settings portal (which is the IP address of the TCD on your network - if you need to connect the TCD to your network, [instructions are here](https://github.com/CircuitSetup/Time-Circuits-Display/wiki/8.-WiFi-Connection-&-TCD-Settings). If it is not connected to a network, the settings can still be changed by going to 192.168.4.1 after connecting to the TCD via WiFi with a phone or laptop) enable "_Use fake power switch_"
3. If using this, you'll probably want to disable the intro by turning off "_Play intro_"
4. Click save to save the settings
5. Provide power to the TCD via the USB port on the bottom of the keypad or via the green connector on the back of the keypad - you should see the white light on the keypad flash
6. Switching on the switch will turn on the TCD display immediately - Turning it off will turn off the display, complete with sound

**Reply by WMasterJedi** [reply]
Thanks mate that was really helpful!
